spec/models/service_template_spec.rb

```diff
@@ -113,6 +113,16 @@
     end
   end
 
+  context "initiator" do
```

Comment:
@mkanoor Do we have any tests for when `:initiator` is not passed or when it is passed as `nil`?  If the key is passed with a `nil` value I think it will save nil in the record instead of using the default value.